source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -691,7 +691,10 @@ module Error = struct
   type kind =
     | FileNotFound
     | FileRead
-    | InvalidJson of string
+    | InvalidJson of {
+        message: string;
+        location: JsonParsing.Location.t;
+      }
```

Comment:
Instead of storing the location in this specific `kind`, could we store the location in `type t` directly? so it's available for all errors. We already store the path there.